src/components/v2/Layout/styles.ts

```diff
@@ -13,5 +13,21 @@ export const useStyles = () => {
         flex-direction: column;
       }
     `,
+    banner: css`
+      margin: ${theme.spacing(4, 12)};
```

Comment:
To be consistent with the rest of the page:
```suggestion
      margin: ${theme.spacing(4, 10, 0)};
```

<img width="134" alt="Screen Shot 2022-06-07 at 17 28 01" src="https://user-images.githubusercontent.com/44675210/172434230-3077cc53-43da-43b6-9362-d1c28b8a50be.png">

With change:
<img width="127" alt="Screen Shot 2022-06-07 at 17 30 20" src="https://user-images.githubusercontent.com/44675210/172434293-3dbd45ba-6de3-4964-a759-a4b52c80b108.png">